A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -204,7 +204,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 
     checkDestData() {
         if (!isFinite(this.perfApprQNH) || !isFinite(this.perfApprTemp) || !isFinite(this.perfApprWindHeading) || !isFinite(this.perfApprWindSpeed)) {
-            this.addTypeTwoMessage("ENTER DEST DATA", "#ffff00", () => {}, () => {
+            this.addTypeTwoMessage("ENTER DEST DATA", "yellow", () => {}, () => {
```

Comment:
This should be amber ("red"), not yellow:
```suggestion
            this.addTypeTwoMessage("ENTER DEST DATA", "red", () => {}, () => {
```

Documentation:

![image](https://user-images.githubusercontent.com/929769/100540607-8f642c00-323e-11eb-9ad9-4dca462aa521.png)